include/abb_librws/v1_0/rw/retcode.h

```diff
@@ -0,0 +1,27 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+#include <abb_librws/rws_retcode.h>
+
+#include <string>
+
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    /**
+     * \brief A function for checking description of diagnostic code.
+     *
+     * https://developercenter.robotstudio.com/api/rwsApi/rwretcode_main_page.html
+     *
+     * \param client RWS client
+     * \param code code to be checked.
+     *
+     * \throw \a RWSError if something goes wrong.
+     */
+     abb::rws::retcode::RetcodeInfo getCode(RWSClient& client, const std::string& code);
```

Comment:
```suggestion
     RetcodeInfo getRetcodeInfo(RWSClient& client, int code);
```
I think `code` should be an `int`, although it in represented as a string in HTTP communication (everything is a string in HTTP).